examples/app/Embeddings.ipynb

```diff
@@ -7,8 +7,36 @@
    "source": [
     "## Embedding explorer\n",
     "\n",
-    "- Embeds a string column using OpenAI Embeddings\n",
-    "- Follows the clustering / projection approach from here: https://umap-learn.readthedocs.io/en/latest/clustering.html"
+    "In this tutorial, we'll create a notebook-based Embeddings explorer. We will use functions, or `Ops`, defined in the weave ecosystem so that they're available to compatible types in the Weave UI.\n",
+    "\n",
+    "You'll learn how to:\n",
+    "- Save and version embeddings\n",
+    "- Build a interactive embeddings explorer using Weave\n",
+    "- Use the weave ecosystem to: \n",
+    "    - Embed a string column using OpenAI Embeddings\n",
+    "    - Cluster & project the embeddings using umap\n",
+    "\n",
+    "Note: we will follow the clustering / projection approach from here: https://umap-learn.readthedocs.io/en/latest/clustering.html"
```

Comment:
oh this is a super useful link I missed / where I might pull the extra description+intuition-building from for the extensions I suggested :)